strings/word_count.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+const cleanString = (input: string): string =>
+  input.trim().replace(/\s+/g, " ");
+
+/** Helper to count words in a given string */
+export function wordCount(input: string): number {
+  if (input.length == 0) return 0;
+
+  const cleanedInput = cleanString(input);
+
+  return cleanedInput.split(" ").length;

```

Comment:
Does `(input.trim().match(/\s+/g) || []).length` also work here?
(Not sure which performs better.)

Reply:
@hayd Yeah it works, but your code is returning the (no. of words - 1). So, incrementing by 1 will result in the correct output. 
I compared both the methods using [Performance API](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now). There is a negligible difference in the performance